packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.scss

```diff
@@ -0,0 +1,192 @@
+@use '~genesys-spark/src/style/focus.scss';
+@use '~genesys-spark/src/style/mixins.scss';
+
+$gux-icon-radio-unchecked: "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3E%3Cpath d='M16 8C16 12.4187 12.4187 16 8 16C3.58125 16 0 12.4187 0 8C0 3.58125 3.58125 0 8 0C12.4187 0 16 3.58125 16 8ZM8 1.5C4.40937 1.5 1.5 4.40937 1.5 8C1.5 11.5906 4.40937 14.5 8 14.5C11.5906 14.5 14.5 11.5906 14.5 8C14.5 4.40937 11.5906 1.5 8 1.5Z' fill-rule='evenodd' clip-rule='evenodd' /%3E%3C/svg%3E";
+$gux-icon-radio-checked: "data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M5 8C5 6.31563 6.31563 5 8 5C9.65625 5 11 6.31563 11 8C11 9.65625 9.65625 11 8 11C6.31563 11 5 9.65625 5 8ZM16 8C16 12.4187 12.4187 16 8 16C3.58125 16 0 12.4187 0 8C0 3.58125 3.58125 0 8 0C12.4187 0 16 3.58125 16 8ZM8 1.5C4.40938 1.5 1.5 4.40938 1.5 8C1.5 11.5906 4.40938 14.5 8 14.5C11.5906 14.5 14.5 11.5906 14.5 8C14.5 4.40938 11.5906 1.5 8 1.5Z' /%3E%3C/svg%3E";
+
+@mixin unchecked {
+  -webkit-mask-image: url($gux-icon-radio-unchecked);
+  mask-image: url($gux-icon-radio-unchecked);
+}
+
+@mixin checked {
+  -webkit-mask-image: url($gux-icon-radio-checked);
+  mask-image: url($gux-icon-radio-checked);
+}
+
+:host(.gux-checked) {
+  .gux-content {
+    border: var(--gse-ui-selectorCard-selected-border-width)
+      var(--gse-ui-selectorCard-selected-border-style)
+      var(--gse-ui-selectorCard-selected-border-color);
+  }
+}
+
+:host(.gux-disabled) {
+  cursor: not-allowed;
+
+  .gux-content {
+    pointer-events: none;
```

Comment:
I think you might also want to add 

```
  -webkit-user-select: none; /* Safari */
  user-select: none;
```

This is just to prevent the selection of text when it is disabled.

Reply:
done